src/components/Footer.tsx

```diff
@@ -38,43 +38,45 @@ const footerLogo = classnames(
   flexDirection('flex-row'),
   space('space-x-4')
 )
+const footerContainer = classnames(inset('bottom-0', 'left-0', 'right-0'))
```

Comment:
Have you tried doing it without `right-0` and `left-0`? Maybe you can use only `right-0`/'left-0` without each other?

Reply:
As I know you can't position [`position: static`](https://developer.mozilla.org/en-US/docs/Web/CSS/position) (which is default for all DOM elements) with `inset, top, bottom etc...`, how does this work?

Reply:
This was an artifact, got rid of it completely thanks for spotting it